src/main.c

```diff
@@ -414,14 +414,16 @@ uint32_t proc_ble(void)
   __ALIGN(4) uint8_t ev_buf[ BLE_EVT_LEN_MAX(BLEGATT_ATT_MTU_MAX) ];
   uint16_t ev_len = BLE_EVT_LEN_MAX(BLEGATT_ATT_MTU_MAX);
 
+  // Init header
+  ble_evt_t* evt = (ble_evt_t*) ev_buf;
+  evt->header.evt_id = 0;
```

Comment:
0 is correct value, though, could you use the `BLE_EVT_INVALID`  instead, which better reflect the invalid event ID.

Reply:
done